package.json

```diff
@@ -30,7 +30,11 @@
     "@commitlint/travis-cli": "12.1.1",
     "check-node-version": "4.0.3",
     "grunt-markdownlint": "2.9.0",
-    "markdownlint": "0.19.0"
+    "grunt-stylelint": "0.16.0",
+    "markdownlint": "0.19.0",
+    "stylelint": "13.13.1",
+    "stylelint-config-standard": "22.0.0",
```

Comment:
This is 2 major versions newer than what we have been using but I looked at the changes and there is nothing breaking or odd. 👍